py4DSTEM/process/diffraction/crystal_ACOM.py

```diff
@@ -1428,160 +1428,108 @@ def match_single_pattern(
         return orientation
 
 
+def orientation_map_to_orix_CrystalMap(self, 
+                                       orientation_map, 
+                                       ind_orientation=0, 
+                                       pixel_size=1.0, 
+                                       pixel_units='px', 
+                                       return_color_key=False
+                                       ):
+    from orix.quaternion import Rotation, Orientation
+    from orix.crystal_map import CrystalMap, Phase, PhaseList, create_coordinate_arrays
+    from orix.plot import IPFColorKeyTSL
+    from diffpy.structure import Atom, Lattice, Structure
```

Comment:
Is this bundled in the install of with orix/pymatgen etc?

Reply:
yes